yabt/builders/python_test.py

```diff
@@ -47,3 +47,14 @@ def test_python_tester_fail(basic_conf):
     populate_targets_graph(build_context, basic_conf)
     with pytest.raises(SystemExit):
         build_context.build_graph(run_tests=True)
+
+@slow
+@pytest.mark.usefixtures('in_tests_project')
+def test_python_tester_flaky(basic_conf):
+    build_context = BuildContext(basic_conf)
+    basic_conf.targets = ['hello_pytest:flaky-test']
+    populate_targets_graph(build_context, basic_conf)
+    target = build_context.targets[basic_conf.targets[0]]
+    target.props.test_env['RANDOM_FILE'] = str(random.randint(0, 20000))
```

Comment:
We generate a random file name to make this test more robust to unexpected terminations

Reply:
We don't want it in /tmp since it is not map (and you can really assume it can be mapped to the host). Using the working directory which ybt always map when running tests solve the problem that the file must exists in two separate docker run commands

Reply:
We don't want it in /tmp since it is not map (and you can really assume it can be mapped to the host). Using the working directory which ybt always map when running tests solve the problem that the file must exists in two separate docker run commands